Remotion/Web/Core/ExecutionEngine/WxeUrlSettings.cs

```diff
@@ -79,9 +79,30 @@ public WxeUrlSettings ()
 
     private WxeUrlSettings (string? urlMappingFile, int maximumUrlLength, string? defaultWxeHandler)
     {
-      UrlMappingFile = urlMappingFile;
+      UrlMappingFile = urlMappingFile ?? string.Empty;
       MaximumUrlLength = maximumUrlLength;
-      DefaultWxeHandler = defaultWxeHandler;
+      DefaultWxeHandler = ValidateDefaultWxeHandler(defaultWxeHandler);
+    }
+
+    private string ValidateDefaultWxeHandler (string? defaultWxeHandler)
```

Comment:
not sure about the method naming. wuldn't expect "Validate" to return a modified value.